data.json

```diff
@@ -70,6 +70,15 @@
     "image": "https://www.aras.com/-/media/Images/Content-Row-Images/im-an-innovator.png",
     "dateAdded": "2018-10-24T11:45:00.000Z",
     "tags": ["clothing", "hacktoberfest", "expired", "stickers"]
+  },
+    {
+    "name": "ArchivesSpace",
+    "difficulty": "medium",
+    "description": "Create PRs to <a href='https://github.com/archivesspace/archivesspace'>ArchivesSpace GitHub repos</a> and the Top COntribuitor will get a ArchivesSpace bag",
+    "reference": "https://community.aras.com/b/english/posts/join-aras-labs-hacktoberfest-2018",
+    "image": "https://archivesspace.org/wp-content/uploads/2016/05/aspacelogo.png",
+    "dateAdded": "2020-04-10T15:21:57.000Z",
```

Comment:
These dates should be formated as YYYY-MM-DD

Reply:
Done!